Minor, initial adjustments for ios-latest page

parlough opened the pull request:
Consider hiding whitespace changes when reviewing.

sfshaza2 reviewed (approved):
Is this a complete redo?

parlough commented:
> Is this a complete redo?

No, but the formatting changes made it look like it. You can "hide whitespace changes" when reviewing to see the actual difference.

A summary of changes:

- For the building entry, changed "Tested. Should just work." to "Tested. Monitoring issues and feedback."
- Removed the placeholder "Cupertino widget A and B" row
- Changed "Tested works." to "Tested and works"
- Removed some inconsistent punctuation
- Fixed the link to the iOS deployment page
